IFC4x3/Properties/i/InsulatedVoltage_1Rmq9PJnfEKuHYQTF1jjZg/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InsulatedVoltage_1Rmq9PJnfEKuHYQTF1jjZg" Name="InsulatedVoltage" UniqueId="5bc34259-4f1a-4e53-8462-69d3c1b6d8ea" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
Should it be renamed to indicate it's a maximum value?

Reply:
Done, commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d